<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AltGui_Alternative_Base_On_Dia</class>
 <widget class="QDialog" name="AltGui_Alternative_Base_On_Dia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>180</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>374</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>374</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Select base measure state...</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../system_sys/source_code/gui/resources/system_sys_resource.qrc">
    <normaloff>:/alt_icon</normaloff>:/alt_icon</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Select the measure state, on which the parameter changing
of one or multiple FPL-section(s) should be based on! 
The height, length and the location of the FPL-section are not affected!</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_base">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>No selection</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>209</width>
         <height>18</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../system_sys/source_code/gui/resources/system_sys_resource.qrc"/>
 </resources>
 <connections/>
</ui>
